files/scripts/lldp.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
This script I think is not getting used unless we update the systemd files. The lldp systemd service files still use the /usr/bin/lldp.sh script.
Additionally I am thinking if we really need this new script, Can we just introduce a new check for check_asic_status() in the current lldp.sh script which exists in sonic ?

Reply:
Had reverted because kvmtest-t0 was failing (since lldp container was not starting). Looks like a separate lldp container runs even in multiasic environment. That could be one reason.

Reply:
Also, it is easier to make changes in the wrapper .sh script. Based on your suggestion, I will have to modify the docker_image_ctl.j2 file to generate the right .sh for lldp, syncd, swss, teamd. Only lldp did not have a wrapper script and I thought this was a good reason to introduce one. Let me know what you think...

Reply:
@judyjoseph Also, when lldp.sh is introduced, I see that kvmtest-t1-lag fails because the container is not created. From the code, do you know if something is incorrect when introducing files/scripts/lldp.sh (/usr/local/bin/lldp.sh) that causes it to fail?

Reply:
@judyjoseph I think we are good now. For some reason the lldp.sh did not have the executable permissions. I re-did a git update-index, and pushed changes and all tests pass.